### PR TITLE
[11.0][MIG] stock_warehouse_orderpoint_stock_info

### PR DESCRIPTION
**Pull request by mpanarin**
migration to Odoo 11

### CODE REVIEW

File: stock_warehouse_orderpoint_stock_info/tests/test_stock_warehouse_orderpoint.py [hunk 1]
```diff
@@ -1,66 +1,51 @@
...
 # Copyright 2016 Serpent Consulting Services Pvt. Ltd.
 # Copyright 2016-17 Eficent Business and IT Consulting Services, S.L.
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from odoo.tests import common
+from odoo.tests.common import SavepointCase
 
 
-class TestStockWarehouseOrderpoint(common.TransactionCase):
+class TestStockWarehouseOrderpoint(SavepointCase):
```
Comment (LoisRForgeFlow) [inline stock_warehouse_orderpoint_stock_info/tests/test_stock_warehouse_orderpoint.py]: Just wondering why you are moving to Savepointcase?

File: stock_warehouse_orderpoint_stock_info/models/stock_warehouse_orderpoint.py [hunk 2]
```diff
@@ -1,48 +1,61 @@
...
-                product = products[order.product_id.id]
-                order.product_location_qty = product['qty_available']
-                order.incoming_location_qty = product['incoming_qty']
-                order.outgoing_location_qty = product['outgoing_qty']
-                order.virtual_location_qty = product['virtual_available']
-
     product_location_qty = fields.Float(
         string='Quantity On Location',
-        compute='_compute_product_available_qty')
+        compute='_compute_product_available_qty'
```
Comment (LoisRForgeFlow) [inline stock_warehouse_orderpoint_stock_info/models/stock_warehouse_orderpoint.py]: If you move to that pro-git-diffs syntax do not forget to use trailing commas.

**Review by LoisRForgeFlow — COMMENTED** [review]
Some minor comments as I see this is mostly a syntax refactor.

File: stock_warehouse_orderpoint_stock_info/tests/test_stock_warehouse_orderpoint.py (hunk 1 above)
Comment (guewen) [reply stock_warehouse_orderpoint_stock_info/tests/test_stock_warehouse_orderpoint.py]: Faster than TransactionCase as the setup can be done only once for all tests (here there's only 1 test but if a test is added, direct benefit)

Comment (LoisRForgeFlow) [reply stock_warehouse_orderpoint_stock_info/tests/test_stock_warehouse_orderpoint.py]: Fair enough! thanks

**Review by max3903 — APPROVED** [review]

**Comment by yvaucher** [comment]
@mpanarin could you squash the translation commits "OCA Transbot updated translations from Transifex"

**Review by yvaucher — APPROVED** [review]
LGTM, please just squash the translation commits and your !squash commit for licenses

**Comment by mpanarin** [comment]
@yvaucher i squashed the commit. Those transifex commits are not actually one after another. Github weirdly shows the tree here. You can see in commits tab

**Comment by yvaucher** [comment]
@mpanarin you can reorder those commits anyway :)

To do this a `git rebase -i` will do. In your editor you can move the commits up and down by cut and paste the line somewhere else.

In this can we shouldn't have conflicts as those commits are updating the same files.
As long as you make sure to keep the same order. 

Then once you have them grouped, you can do the usual squashing.

Here you probably want to move down 

78fecce, 51f54b2 and 773d088

just before

7f86803

With this result:

```
[...]
pick 78fecce
f 51f54b2
f 773d088
f 7f86803
[...]
```

**Review by guewen — APPROVED** [review]

**Pull request merged**